### ISSUE TITLE
Not showing if a code is valid or not.

### ISSUE THREAD

**Issue by KYKKIS2** [Tainted06/Xbox-Code-Checker#3]
Yesterday it was working perfectly but today it won't show me if the code is valid or not nor it wont place it to the txt files.

example:
![Screenshot_5](https://user-images.githubusercontent.com/44586517/205468231-e6e06e70-dedb-4338-981d-27d3ac16064a.png)

**Comment by Tainted06** [comment]
Sorry for the late reply, are your codes in the text files?

**Comment by Tainted06** [comment]
![4dmX3K8JCM](https://user-images.githubusercontent.com/80160851/205651293-9a3fafaa-1591-4e81-9e56-90311b27d994.png)
Tested it and it works fine for me
Double check that you've filled out all the text files and the files are in the correct location.

**Comment by KYKKIS2** [comment]
Yes i added the codes and as you can see on the photo it says 5/5 codes cheched | 100% done at the top bar. 
I don't know what might caused this.

Only when a code is invalid it shows something.

![image](https://user-images.githubusercontent.com/44586517/205699143-17db8c02-d8d5-464f-afb9-f38f8db532a0.png)

Also see that i added 6 codes at 6/6 on the top bar of the program and it only shows the invalid one the others i don't know what.

**Comment by KYKKIS2** [comment]
2 days ago it was working perefect showing ther used the corects and incorect ones

**Comment by KYKKIS2** [comment]
They are USA ONLY TRIAL, does this matter?

**Comment by Tainted06** [comment]
It shouldn't, could you send me a code on discord/telegram for me to test it? (I won't need to redeem it)
Discord: Tainted#0069 
Telegram: @TaintedDev

**Comment by KYKKIS2** [comment]
Yea ill send you now

**Comment by KYKKIS2** [comment]
I just tested it and the USA codes cannot be tested because the WLID of my account is on EUROPE that's why i think.
